dev-toolkit/stop.sh

```diff
@@ -14,6 +14,7 @@ $DOCKER_COMPOSE_CMD \
     --profile schema-registry \
     --profile ksqldb \
     --profile consumer \
+    --profile clientsreduced \
```

Comment:
this must be replaced with

  -f docker-compose.consumer.yaml \
  -f docker-compose.consumer-minimal.yaml \
  
 see 
 
  https://github.com/confluentinc/jmx-monitoring-stacks/blob/main/dev-toolkit/start.sh

Reply:
you mean adding the missing profiles in the stop script, right?

```
--profile consumer-minimal \
--profile schema-registry-primary-secondary \
```